.github/workflows/issues.yml

```diff
@@ -65,3 +58,105 @@
           issue-number: ${{ github.event.issue.number }}
           body: |
             Thanks for opening this issue. The ticket [${{ steps.create.outputs.jira-ticket-id }}](https://jira.mongodb.org/browse/${{ steps.create.outputs.jira-ticket-id }}) was created for internal tracking.
+    reopen_jira_ticket:
```

Comment:
❤️ I missed that feature! Sign we need to start looking into centralized github actions for mongodb org to avoid duplicating similar actions across repositories.

Reply:
totally